src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/MockBuilders/DataSetUploadMockBuilder.cs

```diff
@@ -24,15 +24,12 @@ public DataSetUpload BuildEntity()
             MetaFileName = "meta.data.csv",
             MetaFileSizeInBytes = 157,
             Status =
-                _screenerResult == ScreenerResult.Failed
-                    ? DataSetUploadStatus.FAILED_SCREENING
-                    : DataSetUploadStatus.PENDING_IMPORT,
+                _screenerResult == "Failed" ? DataSetUploadStatus.FAILED_SCREENING : DataSetUploadStatus.PENDING_IMPORT,
```

Comment:
I guess the driver for moving away from enum / constants here and just going with a freeform string was that we don't actually hang any logic off the value anymore?